wialon/api.py

```diff
@@ -134,7 +134,7 @@ def call(self, action_name, *argc, **kwargs):
             params = json.dumps(kwargs, ensure_ascii=False)
 
         params = {
-            'svc': action_name.replace('_', '/', 1),
```

Comment:
This replace was made for purpose, not accidentally. This allows to make requests to generic URLs like `core/search_items` by calling method `core_search_items`. Please, do not change this.

Reply:
following the remote docs why would there be need to have methods like 'core_search_items' yet they have them like core/search_items

Reply:
Exactly, the API URL looks like `core/search_items`. But you cannot name Python function `core/search_items`, slash is forbidden in function names. So you call `Wialon().core_search_items(...)`. Here is wrapper https://github.com/wialon/python-wialon/blob/master/wialon/api.py#L205, which passes args to `call` method, which replaces first underscore with slash. This is definitely a feature, not a bug :slightly_smiling_face: